packages/mds-policy-author/tests/api.spec.ts

```diff
@@ -54,11 +55,8 @@ const request = supertest(ApiServer(api))
 
 const APP_JSON = 'application/json; charset=utf-8'
 
-// TODO
-// change the auth token/authing system so it uses agency_id instead of provider_id
-const AUTH_NON_PROVIDER = `basic ${Buffer.from(`'BOGUS_PROVIDER_ID_TO_BE_REPLACED'`).toString('base64')}`
-const AUTH_PROVIDER_ONLY = `basic ${Buffer.from(`${TEST1_PROVIDER_ID}`).toString('base64')}`
-const AUTH_BAD_PROVIDER_ONLY = `basic ${Buffer.from(`${TEST3_PROVIDER_ID}`).toString('base64')}`
+const SCOPED_AUTH = (...scopes: AccessTokenScope[]) =>
```

Comment:
Any thoughts on what a better place to put this might be? Maybe `mds-test-data`?

Reply:
That would require `mds-test-data` to take a dependency on `mds-api-scopes` which could get weird. Perhaps the definition of `AccessTokenScope` could be moved to `mds-types` and then moving `SCOPED_AUTH` to `mds-test-data` would make sense. Otherwise, it's going to continue to get replicated in every API project.